App.jsx

```diff
@@ -27,56 +23,38 @@ const App = React.createClass({
     this.setState({name: event.target.value});
   },
 
-  handleMessageChange(event) {
-    this.setState({newMessage: event.target.value});
-  },
-
   handleKeyPress(event) {
-    const {name, newMessage} = this.state;
+    const {name} = this.state;
     // If name or newMessage are blank, do not send new message
-    if (!name || !newMessage) {
+    if (!name) {
       return;
     }
 
     // If user hits Enter key, then send message to Firebase database
     // and clear out the message box
-    if (event.key === 'Enter') {
-      Firebase.sendMessage({ name: name, message: newMessage });
-      this.setState({newMessage: ''});
+    if (event.key === '') {
+      // Firebase.sendMessage({name: name, message: newMessage});
     }
   },
 
   renderMessageDiv(message) {
-    return <Message key={message.key} message={message}/>;
+    return <p> </p>;
   },
 
   render() {
     // Iterates through the messages in state to create HTML elements
     // for each message
     const messageDivs = this.state.messages.map(this.renderMessageDiv);
 
-    const {newMessage, name} = this.state;
-
     return (
       <div>
-        <nav className="navbar">
```

Comment:
maybe keep all the `nav`s and `div`s but without the classNames so they can easily type in the style names?